src/directives/popover.js

```diff
@@ -53,8 +54,7 @@ ngeo.popoverDirective = function() {
 
       if (attrs['ngeoPopoverDismiss']) {
         $(attrs['ngeoPopoverDismiss']).on('scroll', function() {
-          ngeoPopoverCtrl.anchorElm.popover('hide');
-          ngeoPopoverCtrl.shown = false;
+          ngeoPopoverCtrl.dismissPopover();
```

Comment:
Does this work? I tested your demo and the popover doesn't dismiss when scrolling the layer tree.